### PR TITLE
Use html body for message sent via email for password change

### PR DESCRIPTION
**Pull request by sharidas**
Use html body for message sent via email
for password change.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

### CODE REVIEW

**Comment by sharidas** [comment]
## Description 
The message sent via email for password success to the user should be sent in html instead of plain text. This change addresses the issue.

## Issue 
- Fixes https://github.com/owncloud/core/issues/35255

## How is it tested
- Create an user with email id
- Access the email to set the password for the user
- Once the password is successful verify the email to see the password change. Here is the screenshot 
passwordchanged

File: lib/Controller/UsersController.php [hunk 1]
```diff
@@ -727,7 +727,7 @@ protected function sendNotificationMail($userId) {
 			$message = $this->mailer->createMessage();
 			$message->setTo([$email => $userId]);
 			$message->setSubject($this->l10n->t('%s password changed successfully', [$this->defaults->getName()]));
-			$message->setPlainBody($msg);
+			$message->setHtmlBody($msg);
```
Comment (micbar) [inline lib/Controller/UsersController.php]: Please add the other template too :-)

Comment (sharidas) [reply lib/Controller/UsersController.php]: Added `setPlainBody` back.

Comment (micbar) [reply lib/Controller/UsersController.php]: 👍

Comment (sharidas) [reply lib/Controller/UsersController.php]: Misunderstood, in my earlier PR. Corrected back to have both plain text and html to make it multipart message.

**Comment by micbar** [comment]
LGTM now.
Lets wait for drone.

File: lib/Controller/UsersController.php (hunk 1 above)
Comment (micbar) [reply lib/Controller/UsersController.php]: done 👍

**Comment by sharidas** [comment]
Backport PR https://github.com/owncloud/core/pull/35260

**Pull request merged**